src/Pages/Withdrawals.razor

```diff
@@ -965,64 +985,72 @@
+                }
 
-            // If the new fee plus the amount exceeds the utxo sum value, show an error
-            // NOTE: revisit this when full rbf because we could add a UTXO
-            var txInfo = await NBXplorerService.GetTransactionAsync(uint256.Parse(_selectedRequest.TxId));
-            var tx = txInfo.Transaction;
-            long vSize = tx.GetVirtualSize();
-            decimal newFee = (vSize * newFeeRate) / 100_000_000m;
+                // If the new fee plus the amount exceeds the utxo sum value, show an error
+                // NOTE: revisit this when full rbf because we could add a UTXO
+                var txInfo = await NBXplorerService.GetTransactionAsync(uint256.Parse(_selectedRequest.TxId));
+                var tx = txInfo.Transaction;
+                long vSize = tx.GetVirtualSize();
+                decimal newFee = (vSize * newFeeRate) / 100_000_000m;
 
-            // Retrieve from database to get the UTXOs
-            _selectedRequest = await WalletWithdrawalRequestRepository.GetById(_selectedRequest.Id);
+                // Retrieve from database to get the UTXOs
+                _selectedRequest = await WalletWithdrawalRequestRepository.GetById(_selectedRequest.Id);
 
-            if (_selectedRequest == null)
-            {
-                ToastService.ShowError("Withdrawal request not found");
-                return;
-            }
+                if (_selectedRequest == null)
+                {
+                    ToastService.ShowError("Withdrawal request not found");
+                    return;
+                }
 
-            if (_selectedRequest.UTXOs != null && _selectedRequest.UTXOs.Count > 0)
-            {
-                var inputAmount = Money.Satoshis(_selectedRequest.UTXOs.Sum(x => x.SatsAmount)).ToDecimal(MoneyUnit.BTC); // Convert to BTC
-                if (inputAmount <= newFee + _selectedRequest.TotalAmount + Constants.BITCOIN_DUST.ToDecimal(MoneyUnit.BTC))
+                if (_selectedRequest.UTXOs != null && _selectedRequest.UTXOs.Count > 0 && _selectedRequest.WalletWithdrawalRequestDestinations!.Count > 1)
                 {
-                    ToastService.ShowError("The new fee plus the amount exceeds the sum of the selected UTXOs or returns dust. Please lower the fee rate.");
+                    var inputAmount = Money.Satoshis(_selectedRequest.UTXOs.Sum(x => x.SatsAmount)).ToDecimal(MoneyUnit.BTC); // Convert to BTC
+                    if (inputAmount < newFee + _selectedRequest.TotalAmount + Constants.BITCOIN_DUST.ToDecimal(MoneyUnit.BTC))
+                    {
+                        ToastService.ShowError("The new fee plus the amount exceeds the sum of the selected UTXOs or returns dust. Please lower the fee rate.");
+                        return;
+                    }
+                }
+
+                // If the request is changeless, show an error
+                // NOTE: Revisit this when full RBF is implemented because we could add a UTXO
```

Comment:
Can you explain me this please?

Reply:
My mistake, I misunderstood a concept from Full RBF